libpod/runtime.go

```diff
@@ -152,6 +157,10 @@ type RuntimeConfig struct {
 	// and all containers and pods will be visible.
 	// The default namespace is "".
 	Namespace string `toml:"namespace,omitempty"`
+	// PauseImage is the image a pod pause container will use to manage namespaces
+	PauseImage string `toml:"pause_image"`
```

Comment:
Should call these PodPauseImage and PodPauseCommand to make it clear where they're used